src/main/resources/blockchain/default-genesis.json

```diff
@@ -0,0 +1,8905 @@
+{
+  "difficulty": "0x0400000000",
+  "extraData": "0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fa",
+  "gasLimit": "0x1388",
+  "nonce": "0x0000000000000042",
+  "ommersHash": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+  "timestamp": "0x0",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
"It's a hash, so it's a number" - hash is an output of hash function, which is a fixed size byte array, not a number. 

"It is something you compute by applying arithmetic transformations" - are you sure `xor`, `and`, `not` are arithmetic operations? ;)

"Same as an Int is a series of 32 bits, and it is a number." - Int is not just a series of 32 bits, it's a series of 32 bits AND an information for JVM to treat these bits as number.

`"abcd"` is also a series of 32 bits, this time with associated information to treat them as characters